gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Aug 16 11:54:01 PDT 2016
+#Thu Mar 02 17:25:18 PST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
The version used here is related to Android Studio upgrade (it was done automatically by Android Studio), I'd prefer making sure we use the gradle version that we use is the recommended one for Android to avoid running into unexpected behavior.